packages/account/src/Configs/get-status-badge-config.tsx

```diff
@@ -3,7 +3,11 @@ import { Link } from 'react-router-dom';
 import { Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 
-const getStatusBadgeConfig = (account_status, openFailedVerificationModal, selected_account_type) => {
+const getStatusBadgeConfig = (
+    account_status: string,
```

Comment:
We can put the possible values for `account_status` like `pending | failed`
We've these as cases in switch bellow

Reply:
I'm not sure, maybe we should skip it now, because I'm not aware of all possible statuses